quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ExportDistTest.java

```diff
@@ -35,6 +35,8 @@ void testExport(KeycloakDistribution dist) {
         cliResult.assertMessage("Export of realm 'master' requested.");
         cliResult.assertMessage("Export finished successfully");
         cliResult.assertNoMessage("Changes detected in configuration");
+        cliResult.assertNoMessage("Listening on:");
+    }
```

Comment:
This line is causing the failure, I think.

Reply:
woops, yes, definitely. will change.

Reply:
@pedroigor CI was green after fixing. I rebased the branch now on the latest changes, should be fine :)